src/packages/media/media/components/input-upload-field/input-upload-field.element.ts

```diff
@@ -65,17 +87,58 @@ export class UmbInputUploadFieldElement extends UmbLitElement {
 		this._extensions = extensions?.map((extension) => `.${extension}`);
 	}
 
+	#getPreviewElementAlias() {
+		const fallbackAlias = this.#previewers.find((preview) => preview.forMimeTypes.includes('*/*'))?.alias;
+
+		const mimeType = this.#getMimeTypeFromPath(this._src);
+		if (!mimeType) return fallbackAlias;
+
+		// Check for an exact match
+		const exactMatch = this.#previewers.find((preview) => {
+			return preview.forMimeTypes.find((type) => type === mimeType);
```

Comment:
I think `.includes` is slightly faster:

```suggestion
			return preview.forMimeTypes.includes(mimeType);
```